http.go

```diff
@@ -1139,12 +1105,12 @@ func (req *Request) onlyMultipartForm() bool {
 // Write doesn't flush request to w for performance reasons.
 //
 // See also WriteTo.
-func (req *Request) Write(w *bufio.Writer) error {
+func (req *Request) Write(w *bufio.Writer) (nn int64, err error) {
```

Comment:
Yeah I wish we could just change our API like that. I really wanted to do this. But this simple change would break a lot of our users code. So I'm afraid we can't do this 😞 

Maybe if we would do a fasthttp 2 and change the import path we could do this. But not for now.

So please revert all these API breaking changes. I suggest maybe just not returning the amount of bytes in your trace functions. `net/http/httptrace` doesn't seem to have this either.

Reply:
it's cool feature. 
and yes, agree to @erikdubbelboer , don't change the API

Reply:
Thanks for the feedback. I was hoping it would be OK since it is just an additional return value and it is really useful to have for logging. I can see why you would not want to break the API though and will update the PR accordingly 👍

Reply:
I updated the PR to remove the backwards incompatible changes.